source/extensions/filters/http/dynamic_forward_proxy/proxy_filter.cc

```diff
@@ -132,26 +132,22 @@ Http::FilterHeadersStatus ProxyFilter::decodeHeaders(Http::RequestHeaderMap& hea
   }
 
   switch (result.status_) {
-  case LoadDnsCacheEntryStatus::InCache: {
+  case LoadDnsCacheEntryStatus::InCache:
```

Comment:
As an aside, I'm guessing if this kind of change is enough to tip us over the line, that things will remain fragile, but fine to remove an exception.

Reply:
If you look at pre-change coverage reports, literally the only lines which did not have coverage were the closing braces in the switch statements and the COVERAGE_EXCLUDE at the bottom.  For some reason closing braces in switch statements are considered non-covered even when (as in this case) the switch code is tested.  Removing the braces removes a coverage report error.